app/index.ejs

```diff
@@ -1,201 +1,379 @@
-  </div>
-</div>
-
-<div ng-if="globalPage.type === 'admin' || globalPage.type === 'landing'">
-  <div class="alert alert-warning growl-message" ng-show="growlMessage">
-    <button type="button" class="close" ng-click="growlMessage=''">&times;</button>
-    <p>{{growlMessage}} <a href="" ng-click="growlUndo()" class="alert-link" translate>Undo</a></p>
-  </div>
-  <header id="masthead">
-    <div class="container">
-      <div class="row">
-        <div class="col-sm-8">
-          <a class="logo" href="/" ng-if="globalPage.type === 'landing'">
-            <i class="fa fa-users"></i> Event Registration Tool
-          </a>
-          <a class="logo" href="eventDashboard" ng-if="globalPage.type === 'admin'">
-            <i class="fa fa-users"></i> Event Registration Tool
-          </a>
-        </div>
-        <div class="col-sm-4 text-right" ng-if="globalUser()">
-          <div class="btn-group usr-greeting spacing-below-xs" uib-dropdown>
-            <button type="button" class="btn btn-default name-btn">Hello <strong ng-bind="globalGreetingName()"></strong></button>
-            <button type="button" class="btn btn-primary dropdown-toggle" uib-dropdown-toggle>
-              <span class="caret"></span>
-              <span class="sr-only">Toggle Dropdown</span>
-            </button>
-            <ul class="dropdown-menu-right" role="menu" uib-dropdown-menu>
-              <li><a href="eventDashboard/"><i class="fa fa-users"></i> <translate>My Dashboard</translate></a></li>
-              <li><a href="help/"><i class="fa fa-medkit"></i> <translate>I Need Help!</translate></a></li>
-              <li><a href="logout"><i class="fa fa-sign-out"></i> <translate>Sign out</translate></a></li>
-            </ul>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <base href="/" />
+
+    <title
+      ng-bind="(currentEventName ? currentEventName + ' - ' : '') + (pageTitle ? (pageTitle  | translate) + ' | ' : '') + ('Event Registration Tool' | translate)"
+    >
+      Event Registration Tool
+    </title>
+
+    <!--[if lt IE 10]>
+      <script type="text/javascript">
+        window.location = '/browserUnsupported.html';
+      </script>
+    <![endif]-->
+
+    <!-- Adobe DTM -->
+    <script>
+      window.digitalData = {
+        user: [
+          {
+            profile: [
+              {
+                profileInfo: {},
+                social: {},
+              },
+            ],
+          },
+        ],
+      };
+    </script>
+    <% if(htmlWebpackPlugin.options.prod) { %>
+    <script src="//assets.adobedtm.com/3202ba9b02b459ee20779cfcd8e79eaf266be170/satelliteLib-928beaae1bce68f8836cc63dd18a9951aa3a1f4e.js"></script>
+    <% } else { %>
+    <script src="//assets.adobedtm.com/3202ba9b02b459ee20779cfcd8e79eaf266be170/satelliteLib-928beaae1bce68f8836cc63dd18a9951aa3a1f4e-staging.js"></script>
```

Comment:
This should have been deleted. We're replacing the DTM/satellite script with the Adobe Launch one. I think your merge kept it around.

Also you deployed everything on staging to master, which in this case was just my branch https://github.com/CruGlobal/conf-registration-web/pull/626. That won't hurt anything being on prod but usually you don't want to do that cuz there could be stuff that isn't ready/is untested.

I probably could have explained it better but usually you leave the branch and the PR alone and just merge  things directly to the staging branch that you need to test. If the staging branch gets to messy I usually reset it to master unless other developers are depending on things being there. And then merge the PRs to master individually.

Reply:
Oh I apologize for this. I have never done a reset to master, is there a simple way too reset to what is currently on master via Github? Or do you do it locally, and then push up to staging?

Reply:
Just locally. I mean you could make a PR with the base set as staging to merge things into staging but that's more work.

Reply:
Everything should be good and up to date now

Reply:
Sweet, thanks for the help Scotty!  :slightly_smiling_face: